test/pkg/consts/const.go

```diff
@@ -0,0 +1,6 @@
+package consts
+
+const (
+	ICEDriverFirmwareVerMinVersion = 4.01
```

Comment:
ICEDriverFirmwareVerMinVersion is another candidate to go into a yaml declarative file.

Reply:
I think we can allow setting the variables in a yaml file in another PR since this is not a blocker for this one.

Reply:
I agree. Also note that we are gonna need to migrate these tests to the emerging testing framework. The yaml file looks more a job to deal in the new testing framework. This is still PoC for SMB deadline..